doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -394,6 +394,8 @@ Modem libraries
 
   * :ref:`sms_readme` library:
 
+    * Fixed time zone handling for received SMSs.
+      The time zone is also now returned in quarters of an hour.
```

Comment:
```suggestion
      The time zone is now returned also in quarters of an hour.
```

Reply:
That's not what I meant. 😄 The time zone is returned only in one format, but there are two changes, a bug fix and the time zone format change. Maybe it should be like this:
```suggestion
      Also, the time zone is now returned in quarters of an hour.
```
Or the word "also" could just be removed, I guess...

Reply:
Oh dear, my bad. Sorry. Maybe it is better without "also".

Reply:
Done.